GenderPayGap.WebUI/Views/ManageOrganisations/UsersRegisteredToReportForOrganisation.cshtml

```diff
@@ -1,35 +1,130 @@
-                </td>
-                <td class="govuk-table__cell">
-                    <a class="govuk-link" href="@(Url.Action("RemoveOrganisation", "Organisation", new {orgId = encryptedOrganisationId, userId = encryptedUserId}))">
-                        Remove user <span class="govuk-visually-hidden">@userRegisteredToReport.Fullname</span>
-                    </a>
-                </td>
-            </tr>
+            Text = "Action",
+            Colspan = 1
         }
-    </tbody>
-</table>
+    };
+    var mobileRows = new List<TableRowViewModel>();
+    @foreach (User userRegisteredToReport in Model.GetFullyRegisteredUsersForOrganisationWithCurrentUserFirst())
+    {
+        var encryptedUserId = Encryption.EncryptQuerystring(userRegisteredToReport.UserId.ToString());
+        var newRow = new TableRowViewModel
+        {
+            Row = new List<TableCellViewModel>
+            {
+                new TableCellViewModel
+                {
+                    Html = @<text>
+                               <p class="govuk-!-margin-top-1 govuk-!-margin-bottom-1">
+                                   @userRegisteredToReport.Fullname
+                                   @if (userRegisteredToReport.UserId == Model.User.UserId)
+                                   {
+                                       <span>(You)</span>
+                                   }
+                                </p>
+                               <p class="govuk-!-margin-top-1 govuk-!-margin-bottom-1">
+                                   <a class="govuk-link" href="@(Url.Action("RemoveOrganisation", "Organisation", new {orgId = encryptedOrganisationId, userId = encryptedUserId}))">
+                                       Remove user <span class="govuk-visually-hidden">@userRegisteredToReport.Fullname</span>
+                                   </a>
+                               </p>
+                            </text>
+                },
+                new TableCellViewModel
+                {
+                    Text = " "
+                }
+            },
+        };
+        mobileRows.Add(newRow);
+    }
+    
+    var desktopRows = new List<TableRowViewModel>();
+    @foreach (User userRegisteredToReport in Model.GetFullyRegisteredUsersForOrganisationWithCurrentUserFirst())
+    {
+        var encryptedUserId = Encryption.EncryptQuerystring(userRegisteredToReport.UserId.ToString());
+        var newRow = new TableRowViewModel
+        {
+            Row = new List<TableCellViewModel>
+            {
+                new TableCellViewModel
+                {
+                    Html = @<text>
+                               @userRegisteredToReport.Fullname
+                               @if (userRegisteredToReport.UserId == Model.User.UserId)
+                               {
+                                   <span>(You)</span>
+                               }
+                               <br/>
+                            </text>

```

Comment:
Can this be wrapped in a `<p>` instead of using `<br/>`? Is that even needed? The next cell is just a naked `<a>` which makes me think it might not be.